examples/largeclusters/swarmmode-vmas.json

```diff
@@ -0,0 +1,97 @@
+{
+  "apiVersion": "vlabs",

```

Comment:
Not sure I understand completely. Did you mean update all agent pool count to 1 from 100? The Swarm Mode templates are similar to K8s, DC/OS, and Swarm template in this regard. I can update it if you like but can you confirm if I understood your comment correctly?